Src/Proccess.cs

```diff
@@ -23,22 +23,23 @@ public static void Start(Data baseData, Data newData, int batchSize)
             do
             {
                 List<Tile> newTiles = newData.GetNextBatch();
-                List<Tile> baseTiles = baseData.GetUpscaledCorrespondingBatch(newTiles);
 
                 tiles.Clear();
                 for (int i = 0; i < newTiles.Count; i++)
                 {
-                    Tile newTile = newTiles[i];
-                    Tile baseTile = baseTiles[i];
-
-                    if (baseTile != null)
+                    var newTile = newTiles[i];
+                    var coords = newTile.GetCoord();
+                    List<CorrespondingTileBuilder> correspondingTileBuilders = new List<CorrespondingTileBuilder>()
                     {
```

Comment:
Remind me please how this works in this situation (logic of what is returned).

Reply:
?